modules/infinite_wishlist/src/Controller/WishlistController.php

```diff
@@ -69,6 +69,7 @@ public function loadProducts(Request $request)
             '#uri' => $variables['uri'],
           ],
           '#provider' => explode("_", $product->product_provider)[0],
+          '#tracking_url' => $product->product_url__uri."&subid=wishlist-".$product->product_id,
```

Comment:
Not sure if we need another variable on the controller as all it does is provide data which can already be fetched in the template (`product_url__uri` and `product_id`). It might be easier to just concatenate in the twig template. 
Also please consider concatenate doing this way: `$product->product_url__uri . '&subid=wishlist-' . $product->product_id` as this is best practice in PHP.